www/js/data.js

```diff
@@ -1,4 +1,5 @@
 var CL_VERSION = 3.0;
+var GS_VERSION = 1.6; // Google Drive Userscript
```

Comment:
I don't think there's much point in this being a float if it just gets converted into a string later.